src/Writer/XLSX/Helper/FileSystemHelper.php

```diff
@@ -329,7 +332,9 @@ public function createContentFiles(Options $options, array $worksheets): self
                 fwrite($worksheetFilePointer, '<sheetPr filterMode="false"><pageSetUpPr fitToPage="false"/></sheetPr>');
                 fwrite($worksheetFilePointer, sprintf('<dimension ref="%s"/>', $range));
             }
-
+            if (isset($pageSetup) && $pageSetup->fitToPage) {
+                fwrite($worksheetFilePointer, '<sheetPr><pageSetUpPr fitToPage="true"/></sheetPr>');
```

Comment:
`sheetPr` tag is already written within the autofilter `if` few lines above: we need to handle the case where both are used

Reply:
From what I understand, when both autofilter and fitToPage are turned on, the fitToPage attribute will be true, right?

Reply:
fixed it to handle the case.